.github/workflows/ci.yml

```diff
@@ -67,3 +71,38 @@ jobs:
         aws-region: ${{ env.AWS_REGION }}
         aws-role-name: ${{ env.AWS_ROLE_NAME }}
         ugbio_base_image: ${{ env.UGBIO_BASE_IMAGE_NAME }}
+
+  ugbio-utils-bioinfo-CI:
+    runs-on: build-docker-4
+    strategy:
+      max-parallel: 5
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          lfs: true
+          path: ${{ env.UGBIO_UTILS_REPO_PATH }}
+
+      - name: Checkout install-bio repo
+        uses: actions/checkout@v4
+        with:
+          repository: Ultimagen/install-bio
+          ref: ${{ env.INSTALL_BIO_REF }}
+          token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
+          fetch-depth: 0
+          path: ${{ env.INSTALL_BIO_PATH }}
+
+      - name: run install-bio.sh
+        run: |
+          bash ${{ env.INSTALL_BIO_PATH }}/${{ env.INSTALL_BIO_SCRIPT }}
+
+      - name: install ugbio-utils venv
+        working-directory: ${{ env.UGBIO_UTILS_REPO_PATH }}
+        run: uv sync --all-extras --all-packages
+
+      - name: run tests
+        working-directory: ${{ env.UGBIO_UTILS_REPO_PATH }}
+        run: |
+          export PATH=/home/linuxbrew/.linuxbrew/bin/:$PATH
```

Comment:
Instead of this you can add the `shell: bash --login -i -eo pipefail {0}` parameter to the step, so that it will happen in a login shell and will load the environment from .bashrc and co.
See [here](https://github.com/Ultimagen/integration/blob/f09442816c22823a3825146e8397ab7292ffd14f/.github/actions/install-bio/action.yml#L91) for an example.

Reply:
The reason I've added this is because in the bioinformatics-install [script](https://github.com/Ultimagen/install-bio/blob/main/install-bio.sh#L219) we install some tools (for example bcftools using brew install) 
somehow here those brew installations are not being recognized - even if I add the `shell: bash --login -i -eo pipefail {0}` and the [tests then fails](https://github.com/Ultimagen/ugbio-utils/actions/runs/13669605073/job/38217185206?pr=107) on bcftools commands

maybe we need to add the 
`export PATH=/home/linuxbrew/.linuxbrew/bin/:$PATH` command to the  ~/.bashrc file in the `install_brew_and_packages` function ?

Reply:
mmm, I see the brew export is added automatically when brew is installed.
Maybe need to have a login shell also for the installation.
Can you try to add the same shell options also in the step that runs `install-bio.sh`? 
You can also try to make it the [default](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/setting-a-default-shell-and-working-directory#example-set-the-default-shell-and-working-directory).

If this doesn't work, I think it is fine to go back to the previous working configuration.

Reply:
I tried but nothing worked
I'm leaving it as it was